app/views/index.html

```diff
@@ -46,6 +46,7 @@
             window.__ROOM_NAME__ = "${roomName}";
             window.__USER_ADMIN__ = "${user.isAdmin()}";
             window.__USER_GUEST__ = "${user.isGuest()}";
+            document.title="${room_name"} + " : breaker";
```

Comment:
Ah, I see. Looks like we can add the room name [here](https://github.com/larvalabs/breaker/blob/master/web/document/DocumentTitle.jsx#L20-L28) pretty easily.  
  
Doing it there will update whenever you change rooms as well.